src/main/resources/templates/pps/penaltyPaid.html

```diff
@@ -5,24 +5,22 @@
       layout:decorate="~{layouts/baseLayout}">
 
 <head>
-    <title th:text="'Late filing penalties for ' + ${companyName}"></title>
+    <title th:text="'Penalties for ' + ${companyName}"></title>
 </head>
 
 <div id="no-payment-found-main-content" layout:fragment="content">
     <form th:action="@{''}" method="post" class="form" id="no-payment-found">
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-two-thirds">
                 <h1 class="govuk-heading-xl">
-                    Late filing penalty for
-                    <span class="govuk-heading-xl" id="company-name" th:text="${companyName}"></span>
+                    This penalty has already been paid
                 </h1>
-                <p class="govuk-body-l">
-                    The penalty with a reference of
-                    <span class="govuk-!-font-weight-bold" th:text="${penaltyNumber}"></span>
-                    has been paid. No further action is required.
+                <p class="govuk-body-l"> Penalty
+                    <span class="govuk-!-font-weight-bold" th:text="${penaltyNumber}"></span> has been paid.
+                <p></p><p class="govuk-body-l">If you have any questions, you can <a href="https://www.gov.uk/contact-companies-house" class="govuk-link">contact us</a>.</p>
```

Comment:
I agree with @klai-ch here, it looks like we should be closing the first paragraph tag just.
It looks like we are opening nested paragraph tags in html - don't think that should be done.
Please can you re-check this to close them and open a new paragraph blocks for the additional sentence.